TEMPLATES/terrascan.toml

```diff
@@ -0,0 +1,14 @@
+# terrascan configuration file https://github.com/accurics/terrascan/blob/master/config/terrascan.toml
+
+# scan and skip rules configuration
+[rules]
+    # scan rules (list of rules to scan, adding rules here will override rules in the policy path)
+    # scan-rules = [
+    #     "AWS.S3Bucket.DS.High.1043",
+    #     "AWS.S3Bucket.IAM.High.0370"
+    # ]
+
+    # skip rules (list of rules to skip)
+    skip-rules = [
+        "AC_AWS_0320"
```

Comment:
Is this in the correct format?

Reply:
@Mark-McAdam - Could you add the SG rule we should be adding here.

Reply:
I initially had a hard time tracking down the rules initially. I discovered this group of rules with reference id. 
We had success adding rules to skip to terrascan.toml by using the reference id, the example that is commented out uses the reference.id 
https://github.com/accurics/terrascan/blob/master/docs/policies/aws.md#aws_security_group

'SSH' (TCP:22) is accessible by a CIDR block range
AWS.ALB.NetworkPortsSecurity.High.0194

ssh port open to internet
AWS.SecurityGroup.NetworkPortsSecurity.Low.0560

One of these might give you the result you are looking for.

Reply:
The terrascan devs themselves [suggest using reference ids](https://github.com/accurics/terrascan/issues/610#issuecomment-867793876), so that was the advice I was going by.